git_cml/__init__.py

```diff
@@ -0,0 +1 @@
+from . import checkpoints, git_utils
```

Comment:
Don't have to do this now but may as well add `__version__ = "0.0.0"` or something here too.